.github/workflows/tests.yml

```diff
@@ -0,0 +1,48 @@
+name: tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+      - cron: '00 4 * * *'  # daily at 4AM
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+
+      fail-fast: false
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: start Kafka and Zookeeper
+      run: docker-compose -f scripts/bitnami-kafka-docker-compose.yml up -d
```

Comment:
I am happy to move that .yml file somewhere else. It would be better all around to have a script to start the docker containers as in sirepo-bluesky and bluesky-kafka.